src/sql/workbench/services/connection/browser/connectionDialogService.ts

```diff
@@ -281,7 +281,7 @@ export class ConnectionDialogService implements IConnectionDialogService {
 						this._connectionManagementService,
 						this._capabilitiesService.getCapabilities(providerName).connection, {
 							onSetConnectButton: (enable: boolean) => this.handleSetConnectButtonEnable(enable)
-						}, providerName);
+						}, providerName, this._inputModel ? this._inputModel.options.authTypeChanged : false);
 			} else {
```

Comment:
You got rid of the bool param here so this shouldn't be necessary anymore right?